app/workers/send_chase_email_to_providers_worker.rb

```diff
@@ -0,0 +1,7 @@
+class SendChaseEmailToProvidersWorker
+  include Sidekiq::Worker
+
+  def perform
+    SendChaseEmailsToProvider.new.call
```

Comment:
As discussed offline, we should add a feature flag here so that we can deploy without the backfil.